rules/windows/credential_access_mimikatz_powershell_module.toml

```diff
@@ -1,20 +1,20 @@
 [metadata]
 creation_date = "2020/12/07"
-maturity = "development"
-updated_date = "2022/05/09"
+maturity = "production"
+updated_date = "2022/05/24"
 
 [rule]
 author = ["Elastic"]
 description = """
 Mimikatz is a credential dumper capable of obtaining plaintext Windows account logins and passwords, along with many
-other features that make it useful for testing the security of networks. This rule detects the Invoke-Mimikatz
-PowerShell command.
+other features that make it useful for testing the security of networks. This rule detects Invoke-Mimikatz PowerShell
+script and alike.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
-language = "eql"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "kuery"
```

Comment:
This rule type change should not be a problem as Development rules are not included on the package, AFAIK.

Reply:
yup - as long as it was never locked (released)